src/utils/prng.ts

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
Is this part of the TSDoc? I've never seen a project use single asteriks to open doc strings before... it doesn't render as expected in my editor (maybe I'm alone there though).

Reply:
It's weird, the linter doesn't complain whether there's one or two asterisks.

Reply:
Right, the TSDoc eslint plugin just enforces _TSDoc_ which is a separate thing from JSDoc. I think we'd still want the JSDoc linter in addition to TSDoc, it seems that the TSDoc one just enforces specific tags, not the style of anything. The documentation for the TSDoc plugin is awful too :confused: 

https://tsdoc.org/pages/packages/eslint-plugin-tsdoc/

Or I guess there's just a single rule that enforces all of it? In any case, it leaves out a lot of stuff that the jsdoc plugin covers that would be nice to have still (like the missing asterisks and stuff)

Reply:
Yeah, the TSDoc plugin seems to only validate the syntax and nothing else. The JSDoc plugin is definitely needed.